scenarios/detection/FAQ.md

```diff
@@ -38,6 +38,13 @@ Annotated object locations are required to train and evaluate an object detector
 
 When creating a new project in VOTT, note that the "source connection" can simply point to a local folder which contains the images to be annotated, and respectively the "target connection" to a folder where to write the output. Pascal VOC style annotations can be exported by selecting "Pascal VOC" in the "Export Settings" tab and then using the "Export Project" button in the "Tags Editor" tab.
 
+For mask (segmentation) annotation, an easy-to-use online tool is
```

Comment:
As before, could you perhaps add a screenshot for mask segmentation and (later or now) also a screenshot for keypoint annotation. thanks!

Reply:
Added